docs/accounts/metrics_parameters.md

```diff
@@ -0,0 +1,42 @@
+3. The response to metrics-flow will be a JSON object that contains the fields `flowId` and `flowBeginTime`. These values will need to be propagated to FxA as query parameters, which can be done using hidden form fields with the names `flow_id` and `flow_begin_time`. You can see an example of how the activity-stream team did this by looking [here](https://hg.mozilla.org/releases/mozilla-beta/diff/5d6261b568c6/browser/extensions/activity-stream/content-src/components/StartupOverlay/StartupOverlay.jsx#l1.22) (link will have to be updated).
+
+## Description of Metrics-Related Query Parameters
+**Note these may not be all the parameters you need to pass for your integration to work!** A more exhaustive but [less detailed list can be found here.](https://github.com/mozilla/fxa-content-server/blob/549fc459b851088ea910da182e17e748fa157f26/docs/query-params.md#context) What is documented below are only those that are relevant for metrics analysis in (e.g.) amplitude.
+
+Notes:
+* The "Regex Used for Validation" column shows the regular expression that the value of each parameter must conform to in order for the measurement to register in our metrics pipeline. If the value does not conform, all events associated with that parameter will fail to be logged.
+
+* You must have access to the mozilla amplitude account to see the example charts. If you are a Mozilla employee, you can email X at mozilla dot org for more information on gaining access to amplitude.
+
+|Name|Description|Example Values|Regex Used for Validation|Amplitude Chart Example|
+|----|-----------|-------------|------------------|-----------------|
+|`entrypoint`|The specific page or browser UI element containing the first step of the FxA sign-in/sign-up process (e.g., enter email form)|`firstrun`| /^[\w.-]+$/|[Firstrun form views](https://analytics.amplitude.com/mozilla-corp/chart/n8cd9no)|
+|`service`|The name of the FxA relier/service that the user is signing into|`sync`|/^(sync&#124;content-server&#124;none&#124;[0-9a-f]{16})$/|[Completed Registrations by Service](https://analytics.amplitude.com/mozilla-corp/chart/85v4c88)|
+|`form_type`|For self-hosted forms only (see above) the type of form that the user submits to begin the FxA flow|either: `email` if the form captures the user's email, otherwise `other`|/^(email&#124;other)$/|NA|
+|`utm_source`|Unambiguous identifier of site or browser UI element that linked to the page containing the beginning of the FxA sign-in/sign-up process |`blog.mozilla.org`|/^[\w.%-]+$/|[Registration form views segmented by utm_source](https://analytics.amplitude.com/mozilla-corp/chart/f5sz7kt)|
+|`utm_campaign`|More general label for the campaign that the site is part of|`firstrun`|/^[\w.%-]+$/|TBD|
+|`utm_content`|Could be used to ID what was clicked to bring user to the page, or to identify the cohort in an A/B test|`textlink`,`firstrun-cohort-a`|/^[\w.%-]+$/|TBD|
```

Comment:
I believe that marketing is currently using utm_term (not utm_content) as the test variation since we weren't using that param actively. (e.g. button_a, button_b)